deepchecks/vision/metrics_utils/metrics.py

```diff
@@ -45,18 +49,58 @@ def get_default_object_detection_scorers() -> t.Dict[str, Metric]:
     }
 
 
+_func_naming_dict = {
+    'precision': 'Precision',
+    'recall': 'Recall',
```

Comment:
I think this are the per class options, right? If so, perhaps for consistency we should call these precision_per_class etc
@Nadav-Barak WDYT?

Reply:
doesn't matter much tbh
default behavior will remain the same
don't mind changing them but not sure if it will make much sense cuz all the name will be longer and all the threshold metrics names won't correspond the the name in that dict